app/src/main/java/org/apache/fineract/data/models/customer/Address.kt

```diff
@@ -2,14 +2,22 @@ package org.apache.fineract.data.models.customer
 
 import android.os.Parcelable
 import com.google.gson.annotations.SerializedName
+import com.raizlabs.android.dbflow.annotation.ConflictAction
+import com.raizlabs.android.dbflow.annotation.PrimaryKey
+import com.raizlabs.android.dbflow.annotation.Table
+import com.raizlabs.android.dbflow.structure.BaseModel
 import kotlinx.android.parcel.Parcelize
+import org.apache.fineract.data.local.database.AppDatabase
 
 @Parcelize
+@Table(name = "Address", database = AppDatabase::class, allFields = true, insertConflict =
+ConflictAction.REPLACE)
 data class Address(
     @SerializedName("street") var street: String? = null,
     @SerializedName("city") var city: String? = null,
+    @PrimaryKey
```

Comment:
You can't make `region` PrimaryKey because more than one customer live in same region.